NitroxClient/MonoBehaviours/Gui/Chat/PlayerChatEntry.cs

```diff
@@ -76,9 +78,17 @@ private void SendMessage()
         {
             if (chat != null && chatMessage.Length > 0)
             {
-                chatBroadcaster.SendChatMessage(chatMessage);
-                ChatLogEntry chatLogEntry = new ChatLogEntry("Me", chatMessage, multiplayerSession.PlayerSettings.PlayerColor);
-                chat.WriteChatLogEntry(chatLogEntry);
+                if(chatMessage.Substring(0, 1) == "/")
+                {
+                    string[] args = { chatMessage.Substring(1) };
+                    NitroxServiceLocator.LocateService<IPacketSender>().Send(new ServerCommand(args));
```

Comment:
Move the `NitroxServiceLocator.LocateService<IPacketSender>()` to the `Awake` method and create a private field to store the IPacketSender to use on this line.